internal/pkg/otel/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: all
```

Comment:
Make sure you include this in mage update target at root.
It would be nice if mage update could regenerate sources. Update go mod versions and then update readme file as it does today.